WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersViewModel.kt

```diff
@@ -101,9 +105,40 @@ class BloggingRemindersViewModel @Inject constructor(
         )
     }
 
-    private fun buildEpilogue(): List<BloggingRemindersItem> {
-        // TODO Add epilogue view items
+    @ExperimentalStdlibApi
+    private fun buildEpilogue(bloggingRemindersModel: BloggingRemindersModel?): List<BloggingRemindersItem> {
+        val numberOfDays = bloggingRemindersModel?.enabledDays?.size ?: 0
+
+        val selectedDays = when {
+            numberOfDays <= THREE_DAYS -> {
+                bloggingRemindersModel?.enabledDays?.map { day ->
+                    day.name.toLowerCase(Locale.ROOT).replaceFirstChar {
+                        it.toUpperCase()
+                    }.plus(" and ")
+                }.toString()
+            }
+            numberOfDays in FOUR_DAYS..SIX_DAYS -> {
+                bloggingRemindersModel?.enabledDays?.map { day ->
+                    day.name.toLowerCase(Locale.ROOT).replaceFirstChar {
+                        it.toUpperCase()
+                    }.take(FIRST_THREE_CHARS).plus(", ")
+                }.toString()
+            }
+            else -> "everyday"
```

Comment:
We can probably merge this `when` block with the `when` block below where we return the actual text and then we wouldn't need to have this `else` branch here.